images/base/scripts/common/setup.sh

```diff
@@ -90,9 +109,14 @@ apt-get install -y build-essential libtool
 #./configure
 ./configure --prefix=/usr
 
-make
-make check
-make install
+if [ x$MACHINE = xs390x ]
+then
+    echo FIXME: protobufs wont compile on 390, missing atomic call
```

Comment:
I should really go fix it.. The protobuf header files did something weird looking for atomic functions... The GCC names are the same as elsewhere, just refused to use them on z.. Either patch inline and/or submit a fix.. Either way I wanted more time to understand the  ifdef spaghetti and it wasn't making tests fail, so I saw that as future item.. Probably should open an issue so that I don't forget